Meter.c

```diff
@@ -124,26 +125,50 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
 
    // First draw in the bar[] buffer...
    int offset = 0;
+   double actualBarWidth=0;
```

Comment:
You should be able to avoid this floating point code by defining `int wsub = w * barsLen;` (rearranging code may apply) and using `wsub` instead of `w` whenever referring to the length of the bar, instead of actual cell position on the screen.